app/controllers/hosts_controller.rb

```diff
@@ -137,6 +137,15 @@ def compute_resource_selected
     end
   end
 
+  def scheduler_hint_selected
+    return not_found unless (params[:host])
```

Comment:
You can save yourself from this by adding this action (scheduler_hint_selected) to `before_action :find_resource` above